_users/cn/develop-with-jax-rs.md

```diff
@@ -89,6 +108,24 @@ ServiceComb支持开发者使用JAX-RS注解，使用JAX-RS模式开发服务。
    </beans>
    ```
 
+* **步骤 4** 添加服务定义。
+
+   在resources目录中添加[microservice.yaml](http://servicecomb.incubator.apache.org/cn/users/service-definition/)。
+
+* **步骤 5** 添加Main启动类
+
+   ```java
+   import org.apache.servicecomb.foundation.common.utils.BeanUtils;
+   import org.apache.servicecomb.foundation.common.utils.Log4jUtils;
+
+   public class Application {
+     public static void main(String[] args) throws Exception {
+        Log4jUtils.init();
```

Comment:
Can you add some comments for these two lines?
BTW, we can use BeanUtils to get the instance of the bean.

Reply:
Done